public/view/header.js

```diff
@@ -0,0 +1,12 @@
+export default () => {
```

Comment:
seria bueno que definan si usan function ó arrow function
```suggestion
export default function() {
```

Reply:
esto para mantener un estándar